src/features/wallet/send/SendTransferForm.tsx

```diff
@@ -0,0 +1,420 @@
+import React, { useEffect, useState } from 'react'
+import { ActivityIndicator } from 'react-native'
+import { useTranslation } from 'react-i18next'
+import { Address } from '@helium/crypto-react-native'
+import { Account } from '@helium/http'
+import Balance, { CurrencyType, NetworkTokens } from '@helium/currency'
+import { useAsync } from 'react-async-hook'
+import animalName from 'angry-purple-tiger'
+import InputField from '../../../components/InputField'
+import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
+import Box from '../../../components/Box'
+import Text from '../../../components/Text'
+import QrCode from '../../../assets/images/qr.svg'
+import Check from '../../../assets/images/check.svg'
+import { useColors } from '../../../theme/themeHooks'
+import LockedField from '../../../components/LockedField'
+import { SendTransfer, SendType } from './sendTypes'
+import { Transfer } from '../../hotspots/transfers/TransferRequests'
+import { decimalSeparator, groupSeparator, locale } from '../../../utils/i18n'
+import { ensLookup } from '../../../utils/explorerClient'
+import useHaptic from '../../../utils/useHaptic'
+import {
+  calculateBurnTxnFee,
+  calculatePaymentTxnFee,
+  calculateTransferTxnFee,
+  useFees,
+} from '../../../utils/fees'
+
+type Props = {
+  account?: Account
+  isLocked: boolean
+  isSeller?: boolean
+  lastReportedActivity?: string
+  onScanPress: () => void
+  sendTransfer: SendTransfer
+  transferData?: Transfer
+  type: SendType
+  updateTransfer: (transferId: number, updates: any) => void
+}
+
+const SendTransferForm = ({
```

Comment:
Thank you for explaining your refactor any why it was necessary. I agree with your reasoning and structure of the components, nice idea to get multiple payment address and amounts supported. 

One thing that this naming is a little confusing with is that this form also handles Hotspot Transfer transaction creation and submission (transferring hotspots between accounts). With your refactor you have made this new object which you call SendTransfers and I think the naming conflicts and makes it a little confusing.

My suggestion is maybe we come up with a new name for these payment / payee pairs. Maybe something like `SendDetailsForm`? Open to other suggestions as well but the Transfer word is the one that makes it confusing.

Then we would call the objects `SendDetails`? Maybe @allenan or @matthewcarlreetz have some input on this naming too.

Reply:
Great suggestion, thank you! I didn't fully realize what the "transfer" was in the context of this Send flow, so now that you explain it, it makes total sense. Just renamed instances of `SendTransfer` to `SendDetails`.